docs/content/reference/versioning-support.md

```diff
@@ -1,42 +1,91 @@
-# Versioning support
-
+# Versioning Support
 There are different components that might require independent versioning and support level:
 
-- API `hypershift.openshift.io`.
-- HyperShift Operator (HO).
-- Control Plane Operator (CPO).
-- HyperShift CLI.
+- Management Cluster
+- API `hypershift.openshift.io`
+- HyperShift Operator (HO)
+- Control Plane Operator (CPO)
+- HyperShift CLI
+- Hosted Control Plane (HCP) CLI
+
+## Support Level
+### Management Cluster
+The upstream HyperShift project does not place strict requirements on the OpenShift version of your management cluster. The HO determines what versions of OCP can be installed through the HostedCluster (HC); see the [HO section](#ho) for more details. However, different versions of the HO are thoroughly tested only on a limited set of OpenShift versions, and this should inform your deployment decisions.
+
+For production use & support, it is required to use a downstream product which bundles a supported build of the HyperShift Operator. This downstream product is called [Multi-Cluster Engine](https://docs.openshift.com/container-platform/4.16/architecture/mce-overview-ocp.html) (MCE) and it is available through OpenShift's OperatorHub. 
+
+MCE versions _do_ require specific OCP versions for the Management Cluster to remain in a supported state. 
+Each version documents its own support matrix. For example, 
+
+- [MCE 2.5](https://access.redhat.com/articles/7056007)
+- [MCE 2.4](https://access.redhat.com/articles/7027079)
+
+As a heuristic, a new release of MCE will run on:
+
+- The latest, yet to be released version of OpenShift
+- The latest GA version of OpenShift
+- Two versions prior to the latest GA version
+
+Versions of MCE can also be obtained with the Advanced Cluster Management (ACM) offering. If you are running ACM, refer to product documentation to determine the bundled MCE version.
+
+The full list of HostedCluster OCP versions that can be installed via the HO on a Management Cluster will depend on the version of the installed HO. However, if you are running a tested configuration or MCE, this list will always include at least (a) the same OCP version as the Management Cluster and (b) Two previous minor versions relative to the Management Cluster. For example, if the Management Cluster is running 4.16 and a supported version of MCE, then the HO will at least be able to install 4.16, 4.15, and 4.14 Hosted Clusters. See the Multi-Cluster Engine section, under the expanded section titled "OpenShift Advanced Cluster Management" on [this page](https://access.redhat.com/support/policy/updates/openshift_operators) for more details. 
```

Comment:
> (a) the same OCP version as the Management Cluster

If we are talking about downstream, that is true. But upstream hypershift project from main branch (4.17) can be installed on a 4.14 OCP mgmt cluster and support versions 4.17,4.16,4.15, and 4.14.